packages/manager/apps/public-cloud/src/index.routes.js

```diff
@@ -8,20 +8,22 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
         .injector()
         .get('publicCloud')
         .getDefaultProject()
-        .then((projectId) =>
-          projectId
+        .then((projectId) => {
+          const $window = trans.injector().get('$window');
+          return projectId
             ? {
                 state: 'pci.projects.project',
                 params: {
                   projectId,
                 },
               }
             : {
-                state: trans.params().onboarding
-                  ? 'pci.projects.onboarding'
-                  : 'pci.projects.new',
-              },
-        ),
+                state:
+                  $window.location.search.search('onboarding') !== -1
```

Comment:
It didn't work with `$transition$.params` ? Do you know why ?

Reply:
No it does not work... :) 
`$transition$.params` has access to the route params. But here, it is a search param outside of angular routes.